### PR TITLE
App form simplify

### PR DESCRIPTION
**Pull request by kailanjian**
## Proposed changes
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. 

## Types of changes

- [x] I've labeled this PR with appropriate labels (`comp/*` component, bug/feature, documentation and other less important ones).

## Checklist
_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging your code._

- [x] I've linked the proper Github Issue(s) to my PR
- [x] Need to internationalize new text
- [x] Doug pls check if mailchimp still works with changes
- [x] Wait for final approval from UX/Marketing on basecamp thread
- [x] **DON'T MERGE UNTIL ALL THESE ARE DONE**

Based on conversation here: https://3.basecamp.com/4474129/buckets/16925967/todos/2893478222#__recording_2913092457

### CODE REVIEW

**Comment by kailanjian** [comment]
image
WIP

File: backend/api/application_form/migrations/0014_auto_20200807_0624.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.0.7 on 2020-08-07 06:24
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```
Comment (JonSEng) [inline backend/api/application_form/migrations/0014_auto_20200807_0624.py:6]: Hi @douglaschan32167 @ktham do you know how we can trigger these for prd safely?

Comment (douglaschan32167) [reply backend/api/application_form/migrations/0014_auto_20200807_0624.py:6]: IIRC go to ECS -> click task definitions -> backend-db-migration -> (choose the one link) -> actions -> Run Task. This assumes that the changes in github have been propagated.

**Comment by douglaschan32167** [comment]
Mailchimp still works the same, both with all of the fields filled out and with only the minimum required fields. Dustin might be sad that there's no data to analyze, but I don't expect it to break based on the few times I tried it.

File: backend/api/application_form/migrations/0014_auto_20200807_0624.py (hunk 1 above)
Comment (JonSEng) [reply backend/api/application_form/migrations/0014_auto_20200807_0624.py:6]: Sweet - @kailanjian if that works could we add that to https://gracepoint.slab.com/posts/ismp-website-infrastructure-overview-17nsiwsj ?

Comment (kailanjian) [reply backend/api/application_form/migrations/0014_auto_20200807_0624.py:6]: Does this mean there's going to be some downtime of the form in between the code change and running the migration?

Comment (douglaschan32167) [reply backend/api/application_form/migrations/0014_auto_20200807_0624.py:6]: Maybe we should split out the migration + models stuff into another PR, push, migrate, then do the FE changes?

**Comment by kailanjian** [comment]
image

latest version, wasn't able to get a final confirmation but hopefully addresses most of our needs

**Pull request merged**